packages/entities/entities-plugins/docs/plugin-form.md

```diff
@@ -83,19 +103,67 @@ The base konnect or kongManger config.
 
 If showing the `Edit` type form, the ID of the plugin.
 
+#### `hideScopeSelection`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+Manually toggle visibility of plugin scope selection control.
+
+#### `isCredential`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+A boolean indicating whether the form is being used to create a plugin or an auth credential.
+
+#### `isWizardStep`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+Will hide the form buttons if you only want to render the form and want to control form submission from the consuming app.
+
+#### `useCustomNamesForPlugin`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+Support instance names for plugins. This can be removed when KHCP-5872-custom-names-for-plugins is removed. Enabled by default for KM.
+
 ### Events
 
 #### error
 
 An `@error` event is emitted when form validation fails. The event payload is the response error.
 
+#### fetch-schema:error
+
+An `@fetch-schema:error` event is emitted when attempting to fetch the plugin schema fails. The event payload is the response error.
+
 #### loading
 
 A `@loading` event is emitted when loading state changes. The event payload is a boolean.
 
 #### update
 
-A `@update` event is emitted when the form is saved. The event payload is the plugin object.
+An `@update` event is emitted when the form is saved. The event payload is the plugin object.
+
+#### model-updated
```

Comment:
question: is this event bound to a `v-model`? If yes, then it needs to be `update:model-value`

Reply:
Nope, this is just the name used everywhere in VFG for the update event to any fields in the form. I understand it's confusing but it's used in so many places throughout the code in both repos I'm hesitant to change it.